src/modules/git_status.rs

```diff
@@ -59,6 +61,8 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
                             format_count(config.ahead, "git_status.ahead", ahead)
                         } else if behind > 0 && ahead == 0 {
                             format_count(config.behind, "git_status.behind", behind)
+                        } else if ahead == 0 && behind == 0 {
+                            format_symbol(config.uptodate, "git_status.uptodate")
                         } else {
                             None
                         }
```

Comment:
I think else is never reached:
```suggestion
                        } else {
                            format_symbol(config.up_to_date, "git_status.up_to_date")
                        }
```